homeassistant/components/sabnzbd/__init__.py

```diff
@@ -50,18 +54,44 @@
 
 SIGNAL_SABNZBD_UPDATED = "sabnzbd_updated"
 
+
+@dataclass
+class SensorEntry(SensorEntityDescription):
+    """Metadata for sensor."""
+
+
 SENSOR_TYPES = {
```

Comment:
With the suggestions below, we can remove the dictionary here, and use a tuple instead.

Reply:
Done

Reply:
Doesn't seem like it, as it is a dictionary at this point

Reply:
I thought I had it done, but pylint do not like it. I get:
E1136: Value 'SensorType' is unsubscriptable
in line 18 of sensor.py
Which makes sense or not ?